lib/player.rb

```diff
@@ -1,8 +1,9 @@
 class Player
```

Comment:
The player class should do more think about what a player does in the TIC-TAC-TOE game and put it here. 
Example: A player selects boxes